<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(49, 27, 146);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>160</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_22">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 90, 254);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_24">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 90, 254);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_21">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 90, 254);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 10</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_23">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 90, 254);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_25">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 90, 254);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 7</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>160</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_29">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 99);
alternate-background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_30">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 99);
alternate-background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_28">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 99);
alternate-background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_26">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 99);
alternate-background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_27">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 99);
alternate-background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>160</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_31">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(38, 65, 143);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 23</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_32">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(38, 65, 143);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 22</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_33">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(38, 65, 143);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 25</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_34">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(38, 65, 143);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 24</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_35">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(38, 65, 143);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 21</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>160</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_38">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(142, 153, 243);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 11</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_37">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(142, 153, 243);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 15</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_36">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(142, 153, 243);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 12</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_39">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(142, 153, 243);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 13</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_40">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(142, 153, 243);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 14</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>160</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_41">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 49, 202);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 19</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_42">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 49, 202);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 16</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_43">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 49, 202);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 20</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_44">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 49, 202);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 18</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_45">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 49, 202);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Pitanje 17</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>221</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QProgressBar" name="progressBar">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="actionLog"/>
    <addaction name="actionPregled"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(103, 70, 195);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLog"/>
   <addaction name="actionPregled"/>
  </widget>
  <action name="actionLog">
   <property name="icon">
    <iconset resource="ikonice.qrc">
     <normaloff>:/slike/loginicon.png</normaloff>:/slike/loginicon.png</iconset>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="actionslika1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>icons8-login-48.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>slika1</string>
   </property>
  </action>
  <action name="actionPregled">
   <property name="icon">
    <iconset resource="ikonice.qrc">
     <normaloff>:/slike/pregled.png</normaloff>:/slike/pregled.png</iconset>
   </property>
   <property name="text">
    <string>Pregled</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ikonice.qrc"/>
 </resources>
 <connections/>
</ui>
